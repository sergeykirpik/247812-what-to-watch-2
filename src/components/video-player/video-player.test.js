import React from 'react';
import renderer from 'react-test-renderer';
import VideoPlayer from './video-player.jsx';

it(`renders correctly`, () => {
  const tree = renderer.create(
      <VideoPlayer
        width={280}
        height={175}
        src="some_preview_link"
        poster="some_poster_image"
        muted={false}
        isPlaying={false}
      />
  ).toJSON();
  expect(tree).toMatchSnapshot();
});
