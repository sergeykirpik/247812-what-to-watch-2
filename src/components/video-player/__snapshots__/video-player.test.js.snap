// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<video
  height={175}
  muted={false}
  poster="some_poster_image"
  width={280}
/>
`;
